### PR TITLE
Fix #2570 : Fixed Typos

### LINKED ISSUE #2570: Fix typos and errors for the following files
1. In KeyboardHelper - there is a typo in `softinputkeybord` (3 changes)
     https://github.com/oppia/oppia-android/blob/74db288b9ac25fb942a8feb35e918bcd7e94e55c/app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt#L7
2. In StartSnapHelper - 'Throws' is not imported (1 change)
    https://github.com/oppia/oppia-android/blob/5fc69fffb97e0c8a0d529238926f3e078cf88b09/app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt#L18
3. In StateAssemblerMarginBindingAdapters - typo in `split-veiw` (1 change)
     https://github.com/oppia/oppia-android/blob/5fc69fffb97e0c8a0d529238926f3e078cf88b09/app/src/main/java/org/oppia/android/app/databinding/StateAssemblerMarginBindingAdapters.java#L9

### PR DESCRIPTION
**Pull request by riturajjain2000**
## Explanation

Fixes #2570 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt [hunk 1]
```diff
@@ -4,13 +4,14 @@ import android.content.Context
...
-/** KeyboardHelper helps to change the visibility of softinputkeybord. */
+/** KeyboardHelper helps to change the visibility of the soft input keyboard. */
 class KeyboardHelper {
   companion object {
     /**
-     * This method hides softinputkeybord
-     * @param view is the input view
-     * @param context context of the activity
+     * This method hides the soft input keyboard.
+     *
```
Comment (FareesHussain) [inline app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt:12]: remove this extra line

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt [hunk 2]
```diff
@@ -20,9 +21,10 @@ class KeyboardHelper {
     }
 
     /**
-     * This method shows softinputkeybord
-     * @param view is the input view
-     * @param context context of the activity
+     * This method shows the soft input keyboard.
+     *
```
Comment (FareesHussain) [inline app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt:25]: remove this extra line

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@riturajjain2000 NIT fixes

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt (hunk 1 above)
Comment (riturajjain2000) [reply app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt:12]: This was suggested by @BenHenning

File: app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt (hunk 2 above)
Comment (riturajjain2000) [reply app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt:25]: This was suggested by @BenHenning previously

**Review by FareesHussain — APPROVED** [review]
LGTM.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by BenHenning — APPROVED** [review]
Thanks @riturajjain2000!

**Comment by riturajjain2000** [comment]
> Thanks @riturajjain2000!

Happy to help!

**Pull request merged**